packages/contracts/contracts/proof-of-use/groups/UserGroups.sol

```diff
@@ -61,7 +61,15 @@ contract UserGroups is OffChainOwnable, TokenTelemetryI, UnionFind {
     UnionFind.unionStruct(UnionFind.findStruct(a), UnionFind.findStruct(b));
   }
 
-  function allowInGroupTransfers(address a, address b) external {
+  // This function is incredibly heavy in terms of gas cost
+  function allowInGroupTransfersAll(address[] members) external {
```

Comment:
what would this function be used for?  Just passing an array of addresses instead of a single one? Why have it if its sort of a foot-gun in terms of gas costs?